spk/transmission/Makefile

```diff
@@ -3,7 +3,7 @@ SPK_VERS = 3.00
 SPK_REV = 19
 SPK_ICON = src/transmission.png
 
-DEPENDS = cross/busybox cross/$(SPK_NAME)
+DEPENDS = cross/$(SPK_NAME)
```

Comment:
When removing cross/busybox the lines below should be removed too.

``` makefile
BUSYBOX_CONFIG = usrmng
ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
```

Reply:
Makes perfect sense.